app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -216,151 +363,149 @@ class DefaultToolbarMenu(
         }
     }
 
-    private val settings = BrowserMenuHighlightableItem(
-        label = context.getString(R.string.browser_menu_settings),
-        startImageResource = R.drawable.ic_settings,
-        iconTintColorResource = if (hasAccountProblem)
-            ThemeManager.resolveAttribute(R.attr.syncDisconnected, context) else
-            primaryTextColor(),
-        textColorResource = if (hasAccountProblem)
-            ThemeManager.resolveAttribute(R.attr.primaryText, context) else
-            primaryTextColor(),
-        highlight = BrowserMenuHighlight.HighPriority(
-            endImageResource = R.drawable.ic_sync_disconnected,
-            backgroundTint = context.getColorFromAttr(R.attr.syncDisconnectedBackground),
-            canPropagate = false
-        ),
-        isHighlighted = { hasAccountProblem }
-    ) {
-        onItemTapped.invoke(ToolbarMenu.Item.Settings)
-    }
+    private val newCoreMenuItems by lazy {
+        val newTabItem = BrowserMenuImageText(
```

Comment:
Instead of making the icon invisible, you could also use SimpleBrowserMenuItem here to remove the icon or do it as a follow up.

Reply:
There is a followup filed for this - good to know which one we should use for that! I'll add it to the issue